src/nvim/ex_docmd.c

```diff
@@ -8411,8 +8421,11 @@ static void ex_redir(exarg_T *eap)
 /// ":redraw": force redraw
 static void ex_redraw(exarg_T *eap)
 {
-  if (State & CMDPREVIEW) {
-    return;  // Ignore :redraw during 'inccommand' preview. #9777
+  // Ignore :redraw during 'inccommand' preview. (#9777)
```

Comment:
extra punctuation is not necessary, the hash symbol is enough. This is the convention of all other comments in the codebase.
```suggestion
  // Ignore :redraw during 'inccommand' preview. #9777
```